frontend/src/components/Pages/FrontPage/MissionOverview/OngoingMissionView.tsx

```diff
@@ -34,18 +36,26 @@ export function OngoingMissionView() {
     const { TranslateText } = useLanguageContext()
     const { ongoingMissions } = useMissionsContext()
     const { installationCode } = useInstallationContext()
+    const [missionsToDisplay, setMissionsToDisplay] = useState<Mission[]>([])
```

Comment:
I think I would like to call this ongoingMissionsToDisplay but also understand if you think it's too big of a name and not justifiable